packages/xo-web/src/xo-app/sr/tab-general.js

```diff
@@ -238,6 +238,26 @@ export default class TabGeneral extends Component {
     }
   )
 
+  _getUrl = group => {
+    const { id, size } = this.props.sr
+    const { baseCopies, vdis, snapshots, usage } = group
+
+    const vdisAndSnapshotsIds = `${vdis.map(_ => _.id).join(' ')} ${map(
+      snapshots,
+      'id'
+    ).join(' ')}`
+
+    const ids =
+      usage > getLimit(size)
+        ? vdisAndSnapshotsIds
+        : (this._lastIds = `${defined(
+            this._lastIds,
+            ''
+          )} ${vdisAndSnapshotsIds} ${map(baseCopies, 'id').join(' ')}`)
```

Comment:
I don't think we should handle the "Others" case from outside `Usage`. We should pass a callback to `Usage` which will be called with a list of items (blocks from the usage graph). And from that list, we compute a relevant URL.